examples/rollup/server.py

```diff
@@ -6,21 +6,22 @@
 
 stripe.api_key = 'sk_test_sample_key'
 
-stripe.api_version = '2022-08-01; embedded_connect_beta=v1'
```

Comment:
did we only want to use the GA api in the sample?

Reply:
Yup! For the beta branch, I will make similar changes but show an example with the components we have in beta instead